lms/djangoapps/mobile_api/users/views.py

```diff
@@ -270,14 +270,18 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
     # the default behavior by setting the pagination_class to None.
     pagination_class = None
 
+    def is_org(self, check_org, course_org):
+        return check_org == None or (check_org.lower() == course_org.lower())
+
     def get_queryset(self):
         enrollments = self.queryset.filter(
             user__username=self.kwargs['username'],
             is_active=True
         ).order_by('created').reverse()
+        org = self.request.query_params.get('org', None)
         return [
             enrollment for enrollment in enrollments
-            if enrollment.course_overview and
+            if enrollment.course_overview and self.is_org(org, enrollment.course_overview.org) and
```

Comment:
Is this making a separate database query for every enrollment to get its related course_overview?  If so, there should be a `select_related` call attached to the queryset above.